### PR TITLE
[MRG] [CLI] improve modelhub cli

### PR DESCRIPTION
**Pull request by HuaizhengZhang**
1. update modelzoo table
install`modelci modelhub list`
dev
image

2. download a model from an url 
install `modelci modelhub get `
dev `python modelci/cli/__init__.py  modelhub get -u https://s3.amazonaws.com/pytorch/models/resnet18-5c106cde.pth -p .cache/resnet18.pth`

3. register a model with a ymal file
install ` modelci modelhub publish`
dev `python modelci/cli/__init__.py  modelhub publish -p example/resnet50_explicit_path.yml`

### CODE REVIEW

**Comment by HuaizhengZhang** [comment]
but the register and convert have bugs and fail to publish models. please refer to #214 

I have not changed any core functions

**Comment by univerone** [comment]
As the database schema has been changed, so is the termianl table, ACCURACY column is no longer avaliable, I'll fix this
Here is a preview
截图_2021-02-05_10-23-07

File: example/resnet50_explicit_path.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-weight: /home/modelci/.modelci/ResNet50/pytorch-torchscript/image_classification/1.zip
+weight: ~/project/ML-Model-CI/.cache/resnet18.pth
```
Comment (univerone) [inline example/resnet50_explicit_path.yml:1]: Maybe the weight name should be  ` resnet50.pth `  as the yml filename suggests

File: example/resnet50_explicit_path.yml [hunk 2]
```diff
@@ -13,4 +13,4 @@ architecture: ResNet50
 framework: PYTORCH
 engine: TORCHSCRIPT
```
Comment (univerone) [inline example/resnet50_explicit_path.yml:14]: engine should be PYTORCH

**Comment by univerone** [comment]
`resnet18-5c106cde.pth`  only stores model weights so this file cannot be used to register and convert.

>     2. download a model from an url
>        install `modelci modelhub get `
>        dev `python modelci/cli/__init__.py  modelhub get -u https://s3.amazonaws.com/pytorch/models/resnet18-5c106cde.pth -p .cache/resnet18.pth`

**Comment by HuaizhengZhang** [comment]
I can initially register a model with our new CLI now. Please refer to #214 

Please approve this and fix the register CLI issue later according to #214 

@univerone @YuanmingLeee

**Pull request merged**